source/Octopus.Server.Client/OctopusAsyncRepository.cs

```diff
@@ -191,7 +193,7 @@ public OctopusAsyncRepository(IOctopusAsyncClient client, RepositoryScope reposi
 
         public async Task<bool> HasLink(string name)
         {
-            var rootDocument = await loadRootResource.Value.ConfigureAwait(false);
+            var rootDocument = await loadRootResource.Value(CancellationToken.None).ConfigureAwait(false);
```

Comment:
default value on token here, to keep stream work manageable. When this class has full support of cancellation token, this will be removed naturally.